src/applications/dhp-connected-devices/containers/App.jsx

```diff
@@ -11,7 +11,7 @@ export default function App() {
       <div className="usa-width-three-fourths">
         <article className="usa-content">
           <div className="schemaform-title">
-            <h1>Connect your health devices</h1>
+            <h1>Connect your health devices to share data</h1>
```

Comment:
Great catch here. Unfortunately, I also caught something in the process. Not sure if we want to be this nit picky, but the wire frames do show that it should say `Connect your health device to share data`. Personally, I think what you all have here makes more sense than the wire frames. Pluralizing it seems more grammatically correct.

Reply:
Agree that what you have with "devices" makes more sense. I'll update the designs to match.

Reply:
Thanks!